hrqb/utils/__init__.py

```diff
@@ -3,15 +3,52 @@
 import datetime
 
 import click
+import pandas as pd
+import us  # type: ignore[import-untyped]
+from dateutil.parser import parse as date_parser  # type: ignore[import-untyped]
 
 
 def today_date() -> datetime.date:
     return datetime.datetime.now(tz=datetime.UTC).date()
 
 
+def normalize_date(date: str | datetime.datetime) -> str | None:
+    if date is None or date == "" or pd.isna(date):
```

Comment:
Minor but [`pd.isna` also returns True for `None`](https://pandas.pydata.org/docs/reference/api/pandas.isna.html#pandas.isna).

Reply:
Nice catch!  Will remove that redundant `is None` clause.